syntax = "proto2";

import "Enum.proto";

option optimize_for = SPEED;

message RoomUserInfo
{
	optional bytes var_name = 1 ;
	optional int64 var_index = 2  ;
	optional bytes picture_uri = 3  ;
}

message RoomInfo2
{
	optional int32 var_id = 1 ;
	optional bytes var_name = 2 ;
	optional int32  var_current_count = 3 ;
	optional int32  var_max_count = 4 ;
}

message Rank
{
	optional int32 var_rank = 1 ;
	optional bytes var_name = 2 ;
	optional int32 var_level = 3;
	optional bytes var_pic_uri = 4 ;
	optional int64 var_index = 5 ;
	optional int32 var_win = 6 ;
	optional int32 var_lose = 7 ;
	optional int32  var_draw = 8 ;
	optional bytes var_contry = 9 ;
}

